### PR TITLE
Add glossary entries for ambient

### PR DESCRIPTION
**Pull request by howardjohn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/reference/glossary/ambient.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Ambient
+test: n/a
+---
+
+Ambient refers to a {{< gloss >}}data plane mode{{< /gloss >}} consisting of a per-node per-namespace component.
+Ambient mode is an alternative to a {{< gloss >}}sidecar{{< /gloss >}} based deployment.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/ambient.md:7]: Glossary entries don't support second level glossary entries, so they need to use links instead.

```suggestion
Ambient refers to a [data plane mode](/docs/reference/glossary/#data-plane-mode) consisting of a per-node per-namespace component.
Ambient mode is an alternative to a [sidecar](/docs/reference/glossary/#sidecar) based deployment.
```
(/docs/reference/glossary/#service)

File: content/en/docs/reference/glossary/dataplane-mode.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+title: Data Plane Mode
+test: n/a
+---
+
+The data plane mode refers to which mode the {{< gloss >}}data plane{{< /gloss >}} is running in.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/dataplane-mode.md:6]:
```suggestion
The data plane mode refers to which mode the [data plane](/docs/reference/glossary/#data-plane) is running in.
```

File: content/en/docs/reference/glossary/hbone.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: HBONE
+test: n/a
+---
+
+HBONE (or HTTP-Based Overlay Network Environment) is a protocol used between Istio components.
+In HBONE, users traffic is securely tunneled over HTTP `CONNECT` tunnels encrypted with {{< gloss >}}Mutual TLS Authentication{{< /gloss >}}.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/hbone.md:7]:
```suggestion
In HBONE, users traffic is securely tunneled over HTTP `CONNECT` tunnels encrypted with [Mutual TLS Authentication](/docs/reference/glossary/#mutual-tls-authentication).
```

File: content/en/docs/reference/glossary/proxyless.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+title: Proxyless
+test: n/a
+---
+
+Proxyless refers to a {{< gloss >}}data plane mode{{< /gloss >}} that runs without proxies by instead
```
Comment (frankbu) [inline content/en/docs/reference/glossary/proxyless.md:6]:
```suggestion
Proxyless refers to a [data plane mode](/docs/reference/glossary/#data-plane-mode) that runs without proxies by instead
```

File: content/en/docs/reference/glossary/sidecar.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+title: Sidecar
+test: n/a
+---
+
+A sidecar, generally, is a container that runs alongside a primary application to provide additional functionality.
+In Istio, sidecar is a {{< gloss >}}data plane mode{{< /gloss >}} that runs an {{< gloss >}}Envoy{{< /gloss >}} proxy alongside each
+{{< gloss >}}Pod{{< /gloss >}}.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/sidecar.md:8]:
```suggestion
In Istio, sidecar is a [data plane mode](/docs/reference/glossary/#data-plane-mode) that runs an [Envoy](/docs/reference/glossary/#envoy) proxy alongside each
[Pod](/docs/reference/glossary/#pod).
```

File: content/en/docs/reference/glossary/waypoint.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+---
+title: Waypoint
+test: n/a
+---
+
+A waypoint refers to the L7 proxy component of {{< gloss >}}ambient{{< /gloss >}} {{< gloss >}}data plane mode{{< /gloss >}}.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/waypoint.md:6]:
```suggestion
A waypoint refers to the L7 proxy component of [ambient](/docs/reference/glossary/#ambient)  [data plane mode](/docs/reference/glossary/#data-plane-mode) .
```

File: content/en/docs/reference/glossary/ambient.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+---
+title: Ambient
```
Comment (craigbox) [inline content/en/docs/reference/glossary/ambient.md:2]: Ambient?  Ambient mesh? 

https://istio.slack.com/archives/C041EQL1XMY/p1677541540860899

File: content/en/docs/reference/glossary/ambient.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+test: n/a
+---
+
+Ambient refers to a [data plane mode](/docs/reference/glossary/#data-plane-mode) consisting of a per-node per-namespace component.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/ambient.md:6]: Is this better?
```suggestion
Ambient refers to a [data plane mode](/docs/reference/glossary/#data-plane-mode) consisting of a per-node and optionally a per-namespace component.
```

File: content/en/docs/reference/glossary/dataplane-mode.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+---
+title: Data Plane Mode
+test: n/a
+---
+
+The data plane mode refers to which mode the [data plane](/docs/reference/glossary/#data-plane) is running in.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/dataplane-mode.md:6]: Istio currently supports three modes: [sidecar](/docs/reference/glossary/#sidecar), [ambient](/docs/reference/glossary/#ambient), and [proxyless](/docs/reference/glossary/#proxyless).

Comment (frankbu) [inline content/en/docs/reference/glossary/dataplane-mode.md:6]:
```suggestion
The data plane mode refers to which mode the [data plane](/docs/reference/glossary/#data-plane) is running in.

Istio currently supports three modes: [sidecar](/docs/reference/glossary/#sidecar), [ambient](/docs/reference/glossary/#ambient), and [proxyless](/docs/reference/glossary/#proxyless).
```

File: content/en/docs/reference/glossary/waypoint.md [hunk 10]
```diff
@@ -0,0 +1,7 @@
+---
+title: Waypoint
+test: n/a
+---
+
+A waypoint refers to the L7 proxy component of [ambient](/docs/reference/glossary/#ambient) [data plane mode](/docs/reference/glossary/#data-plane-mode).
```
Comment (frankbu) [inline content/en/docs/reference/glossary/waypoint.md:6]:
```suggestion
A waypoint refers to the L7 proxy component of [ambient](/docs/reference/glossary/#ambient) data plane mode.
`

File: content/en/docs/reference/glossary/ztunnel.md [hunk 11]
```diff
@@ -0,0 +1,7 @@
+---
+title: ztunnel
+test: n/a
+---
+
+Ztunnel refers to the node proxy component of [ambient](/docs/reference/glossary/#ambient) [data plane mode](/docs/reference/glossary/#data-plane-mode).
```
Comment (frankbu) [inline content/en/docs/reference/glossary/ztunnel.md:6]:
```suggestion
Ztunnel refers to the node proxy component of [ambient](/docs/reference/glossary/#ambient) data plane mode.
```

File: content/en/docs/reference/glossary/ztunnel.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
...
+Ztunnel runs on each node and securely transmits traffic using the {{< gloss >}}HBONE{{< /gloss >}} protocol.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/ztunnel.md:7]:
```suggestion
Ztunnel runs on each node and securely transmits traffic using the [HBONE](/docs/reference/glossary/#HBONE) protocol.
```

File: content/en/docs/reference/glossary/waypoint.md [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+A waypoint runs on a per-namespace basis and handles all traffic entering that namespace.
```
Comment (kyessenov) [inline content/en/docs/reference/glossary/waypoint.md:7]: per namespace or per service account

File: content/en/docs/reference/glossary/waypoint.md (hunk 10 above)
Comment (kyessenov) [inline content/en/docs/reference/glossary/waypoint.md:6]: L7 is confusing with a  tunnel being L7 based. Suggest to call it simply HTTP: "to the HTTP proxy component"

File: content/en/docs/reference/glossary/hbone.md [hunk 14]
```diff
@@ -0,0 +1,7 @@
+---
+title: HBONE
```
Comment (kyessenov) [inline content/en/docs/reference/glossary/hbone.md:2]: Without a good definition, throwing new terms like HBONE is more confusing than clarifying IMHO.

File: content/en/docs/reference/glossary/hbone.md [hunk 15]
```diff
@@ -0,0 +1,7 @@
...
+test: n/a
+---
+
+HBONE (or HTTP-Based Overlay Network Environment) is a protocol used between Istio components.
```
Comment (frankbu) [inline content/en/docs/reference/glossary/hbone.md]:
```suggestion
HBONE (or HTTP-Based Overlay Network Environment) is a secure tunneling protocol used between Istio components.
```

**Review by frankbu — APPROVED** [review]

**Pull request merged**